doc/man/dnsviz-print.1

```diff
@@ -296,13 +296,13 @@ warnings are present
 
 .P
 For example, a DNSKEY with warnings is shown as: [.?]  A DS corresponding to a
-non-existent DNSKEY is represented as: [-].
+nonexistent DNSKEY is represented as: [-].
 
 RRSIGs are shown below the RRset they cover, indented from the RRset.
 
 .SS Negative Responses
 If a response is negative, then the appropriate "NODATA" or "NXDOMAIN" text is
-shown adjacent the type queried, e.g., "A: NXDOMDAIN".  If there was an SOA
+shown adjacent the type queried, e.g., "A: NXDOMAIN".  If there was an SOA
```

Comment:
I'm not _entirely_ certain about this change

Reply:
This one is fine.